modules/mono/editor/GodotTools/GodotTools/Build/MsBuildFinder.cs

```diff
@@ -161,8 +161,21 @@ private static string FindMsBuildToolsPathOnWindows()
 
             // Try to find 15.0 with vswhere
 
-            string vsWherePath = Environment.GetEnvironmentVariable(Internal.GodotIs32Bits() ? "ProgramFiles" : "ProgramFiles(x86)");
-            vsWherePath += "\\Microsoft Visual Studio\\Installer\\vswhere.exe";
+            var envNames = Internal.GodotIs32Bits() ? new[] { "ProgramFiles", "ProgramW6432" } : new[] { "ProgramFiles(x86)", "ProgramFiles" };
+
+            string vsWherePath = null;
+            foreach (var envName in envNames)
+            {
+                vsWherePath = Environment.GetEnvironmentVariable(envName);
+                if (!string.IsNullOrEmpty(vsWherePath))
+                {
+                    vsWherePath += "\\Microsoft Visual Studio\\Installer\\vswhere.exe";
```

Comment:
Not directly about this PR's changes, but can we just use `/` for the directory separator on Windows in the C# code? Windows has supported `/` as a directory separator for decades now, and I think all of the tooling that Godot uses supports it.